styleguide/source/_patterns/01-molecules/12-ec/02-ec-bundle-card/_ec-bundle-card.scss

```diff
@@ -0,0 +1,101 @@
+.ec-bundle-card {
```

Comment:
At the beginning our goal was to name patterns/classes/everything according to BEM when possible. So this would be something like `.card-course-bundle`. I was confused by the class name in this PR because `.ec-bundle-card` wasn't really descriptive enough - at the very least I'd recommend changing this to `.ec-course-bundle-card`. 

Again, I'm okay with merging this in and fixing it later when we reorganize how cards are organized, but just pointing it out for discussion purposes.